### PR TITLE
Brandon/runnable app

### PR DESCRIPTION
**Pull request by brosscode**
Adds a basic runnable app module.

### CODE REVIEW

File: appLegacy/build.gradle [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    implementation 'com.google.android.material:material:1.8.0'
+    implementation 'androidx.constraintlayout:constraintlayout:2.1.4'
+    testImplementation 'junit:junit:4.13.2'
+    androidTestImplementation 'androidx.test.ext:junit:1.1.5'
+    androidTestImplementation 'androidx.test.espresso:espresso-core:3.5.1'
+
+    // Hilt
+    implementation "com.google.dagger:hilt-android:2.44"
+    kapt "com.google.dagger:hilt-compiler:2.44"
+}
```
Comment (ForzenAcademy) [inline appLegacy/build.gradle:50]: Erase the empty file below this in the change list

File: appLegacy/proguard-rules.pro [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# Add project specific ProGuard rules here.
```
Comment (ForzenAcademy) [inline appLegacy/proguard-rules.pro]: Erase this

File: appLegacy/src/main/res/xml/data_extraction_rules.xml [hunk 3]
```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?><!--
```
Comment (ForzenAcademy) [inline appLegacy/src/main/res/xml/data_extraction_rules.xml]: Erase

File: appLegacy/src/main/res/xml/backup_rules.xml [hunk 4]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?><!--
```
Comment (ForzenAcademy) [inline appLegacy/src/main/res/xml/backup_rules.xml]: Erase

**Review by ForzenAcademy — APPROVED** [review]
READ THE COMMENTS FIRST AND ADDRESS THEM BEFORE MERGING OR I SWEAR TO GOD

File: appLegacy/build.gradle (hunk 1 above)
Comment (brosscode) [reply appLegacy/build.gradle:50]: Deleted the consumer-rules.pro

File: appLegacy/proguard-rules.pro (hunk 2 above)
Comment (brosscode) [reply appLegacy/proguard-rules.pro]: Deleted

File: appLegacy/src/main/res/xml/backup_rules.xml (hunk 4 above)
Comment (brosscode) [reply appLegacy/src/main/res/xml/backup_rules.xml]: Deleted

File: appLegacy/src/main/res/xml/data_extraction_rules.xml (hunk 3 above)
Comment (brosscode) [reply appLegacy/src/main/res/xml/data_extraction_rules.xml]: Deleted

**Pull request merged**